packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -109,6 +114,14 @@ const {
   }
 }))
 
+const { result: workspacesInvitesResult } = useQuery(
+  projectsDashboardWorkspaceInvitesQuery,
+  undefined,
+  () => ({
+    enabled: isWorkspacesEnabled
```

Comment:
`isWorkspacesEnabled.value` ?

Reply:
@cdriesler `enabled` accepts refs too, but better to be consistent anyway - so ive pushed a change